funcbench/README.md

```diff
@@ -0,0 +1,53 @@
+## Intro
+A tool used as a github action to run a `go test -bench` and compare changes from a PR against another branch. The benchmark is triggered by creating a comment which specifies a branch to compare. The results are then posted back as a PR comment. 
+Comparison is done using [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp). Arguments for the benchcmp are read from files created by previous action (for example [commentMonitor](/tools/commentMonitor/main.go)), which is responsible for the comment parsing.
+
+## Use
+Tests are triggered by posting a comment in a PR with the following format:
+`/funcbench <branch> <golang test regex>  [-no-race]`
+Specifying which tests to run are filtered by using the standard golang regex format. 
+By default all benchmarks run with `-race` flag enabled and it can be disabled by appending `-no-race` at the end of the comment. 
+After a successful run a github comment is created with the results.
+
+### Example Github actions workflow file
```

Comment:
not sure if we should keep this example file or just put a link to the one that will be in the prometheus repo.
Having it here means that when we change things need to also update the example where if we just put a link to the prometheus repo it will always be up to date.

Reply:
The example, is just an example. As long as it is a valid example I think it is okay, and it does not have to be directing to something useful. I think we should modify the example only if we modify the `funcbench`. Also, it might be possible that the link will be broken and we might never find out. So I would rather keep it as in the README as a text.

Reply:
ok